### PR TITLE
feat: Make log.meta optional

### PR DESCRIPTION
**Pull request by kruczyna**
Currently the log.meta property is not optional, this PR aims to make it optional, from:
```    const meta = Object.assign(
      log.meta, // Optional custom property to add more meta to the LogEntry
      { severity: this.mapSeverity(log.level) }
    )
```
to:
```
    const meta = log.meta
      ? Object.assign(log.meta, { severity: this.mapSeverity(log.level) })
      : Object.assign(
        { severity: this.mapSeverity(log.level) }
      )
```
+ added unit tests

### CODE REVIEW

File: lib/cloud-logging.js [hunk 1]
```diff
@@ -80,10 +80,11 @@ class CloudLogging {
...
 
-    const meta = Object.assign(
-      log.meta, // Custom property to add more meta to the LogEntry
-      { severity: this.mapSeverity(log.level) }
-    )
+    const meta = log.meta
+      ? Object.assign(log.meta, { severity: this.mapSeverity(log.level) })
+      : Object.assign(
+        { severity: this.mapSeverity(log.level) }
+      )
```
[the remark this answers is not in the document]
Comment (kruczyna) [reply lib/cloud-logging.js]: Ah true! Removed in: [Remove unnecessary Object.assign statement](https://github.com/metcoder95/cloud-pine/pull/16/commits/f34d1c0596223af97170e1d181b525211ee924b3)

File: lib/cloud-logging.js [hunk 2]
```diff
@@ -80,10 +80,11 @@ class CloudLogging {
       return
     }
 
-    const meta = Object.assign(
-      log.meta, // Custom property to add more meta to the LogEntry
-      { severity: this.mapSeverity(log.level) }
-    )
+    const meta = log.meta
```
[the remark this answers is not in the document]
Comment (kruczyna) [reply lib/cloud-logging.js]: Applied in:
[Apply meta type check](https://github.com/metcoder95/cloud-pine/pull/16/commits/7f2cb681a373f460c791d859ec0d20b49035d065) and [Add unit tests covering empty strings](https://github.com/metcoder95/cloud-pine/pull/16/commits/eaefa3a38897d26e1fe6626877667d012046cc0b) 
WDYT now? :)

[the remark this answers is not in the document]
Comment (metcoder95) [reply lib/cloud-logging.js]: LGTM 👌

**Review by metcoder95 — APPROVED** [review]
